game/script-topics.rpy

```diff
@@ -294,22 +312,283 @@ label talk_get_picture_permission:
             $ natsuki_approves = random.randint(1, 100) <= ((100 - bad_screenshot_streak) * 25)
```

Comment:
```suggestion
            $ natsuki_approves = random.randint(1, 100) <= ((100 - jn_screenshots.bad_screenshot_streak) * 25)
```

bad_screenshot_streak wasn't accessible by itself

Reply:
```suggestion
            $ natsuki_approves = random.randint(1, 100) <= (100 - (bad_screenshot_streak * 25))
```
Actually I also wonder if this wasn't the intended formula

Reply:
My bad! I remember fiddling with this formula a bit, and must have misplaced some brackets when tidying up after verifying it worked. Thank you!